modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/DropwizardServerGenerator.scala

```diff
@@ -334,6 +333,11 @@ object DropwizardServerGenerator {
                   // the @NotNull annotation *must* come before the @FormDataParam annotation.  See:
                   // https://github.com/eclipse-ee4j/jersey/issues/3632
                   parameter.getAnnotations.add(0, new MarkerAnnotationExpr("NotNull"))
+
+                  // Vavr's validation support for some reason requires this.
+                  if (param.param.getTypeAsString.startsWith("io.vavr.collection.")) {
```

Comment:
Does this impact folks that use `x-java-type: ...` to specify a shorthand vavr type?

Reply:
Yes and no.  If they override it to a `java.util` type, then this check will not match, and all will be well.  If they use a different `io.vavr` type, this check should still match, and all still should be well.

Reply:
To answer the question I maybe didn't realize you are maybe asking: if people are _already_ overriding to an `io.vavr` type, I suspect it does not work, because you get a runtime failure during validation without this extra annotation.